pyuvdata/uvbeam/tests/test_uvbeam.py

```diff
@@ -786,7 +681,7 @@ def test_power_spatial_interpolation():
         + np.arange(0, 2 * np.pi, np.pi / 9.0).tolist()
     )
     za_interp_vals = np.array(
-        (np.zeros((18)) + np.pi / 4).tolist() + (np.zeros((18)) + np.pi / 12).tolist()
+        (np.zeros((18)) + np.pi / 18).tolist() + (np.zeros((18)) + np.pi / 36).tolist()
```

Comment:
Just curious: why did these values change? (I don't think we need to make a comment, just a personal curiosity...)

Reply:
because the size of the beam on the sky changed (I cut it down to only cover 10 degrees in zenith angle). The old values were for regions outside what is now cut and the interpolation errors.

Reply:
Sounds good, thanks for the explanation.